src/services/webclient.ts

```diff
@@ -1637,16 +1637,18 @@ export class WebClientService {
                                     if (identity !== this.me.id) {
                                         // tslint:disable-next-line: no-shadowed-variable
                                         const contact = this.contacts.get(identity);
-                                        if (contact === undefined
-                                            || !hasFeature(contact, requiredFeature, this.$log)) {
+                                        if (contact === undefined) {
+                                            this.$log.error(`Cannot retrieve contact ${identity}`);
+                                        } else if (!hasFeature(contact, requiredFeature, this.$log)) {
```

Comment:
This is the main change. The rest is just code formatting.

Reply:
is it okay that an undefined contact is now a *supported member*?

Reply:
In that case, can you add a note why you've made the change? Otherwise if I'd read that code in a few months time...

Reply:
I like comments for code sections that are really weird and I consider this to be one of the considerably weird conditions. :wink: